NumLib/Fem/ShapeFunction/ShapeTet10-impl.h

```diff
@@ -20,9 +20,9 @@ void ShapeTet10::computeShapeFunction(const T_X &r, T_N &N)
     N[4] = 4.0 * r[0] * (1.0 - r[0] - r[1] - r[2]);
     N[5] = 4.0 * r[0] * r[1];
     N[6] = 4.0 * r[1] * (1.0 - r[0] - r[1] - r[2]);
-    N[7] = 4.0 * r[0] * r[2];
```

Comment:
i see. thanks. in that case, the shape function should support the VTK ordering. Otherwise its results could be wrong, since current ogs6 is following VTK's element definitions.